src/main/java/org/topbraid/shacl/tools/Infer.java

```diff
@@ -43,15 +43,51 @@ public static void main(String[] args) throws IOException {
 		System.setErr(oldPS);
 		infer.run(args);
 	}
-	
-	
+
+
 	private void run(String[] args) throws IOException {
 		Model dataModel = getDataModel(args);
 		Model shapesModel = getShapesModel(args);
 		if(shapesModel == null) {
 			shapesModel = dataModel;
 		}
-		Model results = RuleUtil.executeRules(dataModel, shapesModel, null, null);
-		results.write(System.out, FileUtils.langTurtle);
-	}
+        int maxIterations = getMaxIterations(args);
```

Comment:
When you look at the following lines in the diff you should see on GitHub that the indentation/tabs is different. Would be nice if you could align this.

I believe the rest looks good so if you update these I could merge it in.

Reply:
Formatting should be fixed now! I forgot to apply it to both files